app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title><%= content_for(:title) || "Online Classroom" %></title>
```

Comment:
Let's add the i18n gem and move the strings there

Reply:
Made the changes apu. Thank you!